core/domain/question_services.py

```diff
@@ -85,31 +86,117 @@ def _create_new_question(committer_id, question, commit_message):
     create_question_summary(question.id, committer_id)
 
 
-def create_new_question_skill_link(question_id, skill_id, skill_difficulty):
-    """Creates a new QuestionSkillLink model.
+def create_multi_question_skill_links_for_question(
+        user_id, question_id, skill_ids, skill_difficulties):
+    """Creates multiple new QuestionSkillLink models. Also adds
+    the skill ids to the linked skill ids of the Question model.
 
     Args:
+        user_id: str. ID of the creator.
+        question_id: str. ID of the question linked to the skill.
+        skill_ids: list(str). ID of the skills to which the question is linked.
+        skill_difficulties: list(float). The difficulty of the question with
+            respect to a skill, represented by a flot between
+            0 and 1 (inclusive).
+
+    Raises:
+        Exception. Skill difficulties and skill ids should match. The lengths of
+        the two lists are different.
+    """
+    if len(skill_ids) != len(skill_difficulties):
+        raise Exception(
+            'Skill difficulties and skill ids should match. The lengths of the '
+            'two lists are different.')
+    question = get_question_by_id(question_id)
+    if not question:
+        raise Exception('Question doesn\'t exist in the backend')
+
+    new_question_skill_link_models = []
+    for index, skill_id in enumerate(skill_ids):
+        new_question_skill_link_models.append(
+            question_models.QuestionSkillLinkModel.create(
+                question_id, skill_id, skill_difficulties[index]))
+
+    new_linked_skill_ids = copy.deepcopy(question.linked_skill_ids)
+    new_linked_skill_ids.extend(skill_ids)
```

Comment:
I think currently there is a restriction about at most three skills linked to one question. Should we check it here as well?

Reply:
Can you point me to this restriction? Because that means we have to do this while creating question_skill_links as well

Reply:
https://github.com/oppia/oppia/blob/develop/core/domain/question_services.py#L350

In my PR I also set "MAX_SKILLS_PER_QUESTION": 3

Reply:
That line of code is different, that is restricting the number of skill_ids for which you can fetch questions. However, a question can be linked to more than 3 skill ids. About your MAX_SKILLS_PER_QUESTION, where is it being used in the PR?